src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,190 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+
```

Comment:
process this field during `add` and `remove` operations
```suggestion
        int size;
```

Reply:
not fixed